BaseFilter/sections/css_extended.txt

```diff
@@ -1943,6 +1943,7 @@ indiastudychannel.com#?#.main > div.row:has(> div.col-md-12 > div.item > div.row
 indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div#HP-gpt-passback)
 indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div[id^="div-gpt-ad-"])
 !
+flasharch.com#?#div[class*="MuiGrid-grid-lg-"] ~ div[class*="css-"]:has(.adsbygoogle)
```

Comment:
Looks bit wide.

Reply:
Is there other leftover?

Reply:
I mean may possibly cause false positive.

Reply:
OK. I will change it.
Confirmed: https://flasharch.com/boards/5/read/749